config/environments/development.rb

```diff
@@ -96,4 +96,10 @@
     end
   end
 
+  #Initialize Payment Gateway with valid credentials
+  ActiveMerchant::Billing::Base.mode = :test
+  ::GATEWAY = ActiveMerchant::Billing::StripeGateway.new(:login => ENV['SECRET_KEY'])
```

Comment:
This env variable could have a more specific name. `OSEM_STRIPE_SECRET_KEY` ?
Also, maybe it should be added to `dotenv.example`